config/samples/primehub_v1alpha1_phdeployment.yaml

```diff
@@ -2,30 +2,30 @@ apiVersion: primehub.io/v1alpha1
 kind: PhDeployment
 metadata:
   name: user-defined-postfix
-  namespace: hub
+  namespace: seldon-system
```

Comment:
can we change this to hub?

Reply:
sure, I just tested that seldon can be run in hub namespace